docs/source/user-guide/getting-started.rst

```diff
@@ -113,50 +114,41 @@ If a newer version of conda is available, type ``y`` to update:
 Managing environments
 =====================
 
-Conda allows you to create separate environments containing files, packages,
-and their dependencies that will not interact with other environments.
+Conda allows you to create separate environments, each containing their own files, packages,
+and package dependencies. The contents of each environment do not interact with one another.
 
 When you begin using conda, you already have a default environment named
-``base``. You don't want to put programs into your base environment, though.
-Create separate environments to keep your programs isolated from each other.
+``base``. **Don't install programs into your base environment.** Instead,
+create separate environments for each project to keep your programs isolated from each other.
 
 #. Create a new environment and install a package in it.
 
    We will name the environment ``snowflakes`` and install the package
-   BioPython. At the Anaconda Prompt or in your terminal window, type
-   the following:
+   BioPython:
 
    .. code::
 
       conda create --name snowflakes biopython
 
-   Conda checks to see what additional packages ("dependencies")
-   BioPython will need, and asks if you want to proceed:
+   Conda checks to see what additional packages
+   BioPython will need to run (BioPython's dependencies) and lists them for you.
+   Type ``y`` and press Enter to proceed:
 
    .. code::
 
       Proceed ([y]/n)? y
 
-   Type "y" and press Enter to proceed.
-
-#. To use, or "activate" the new environment, type the following:
```

Comment:
Removed this and the items below to try and streamline the getting started. It seemed unnecessary to have info on such an old version of conda anymore, but if we wanted to keep the information on `activate <ENV>` and `source activate <ENV>`, I could make a specific page on it.